src/xrGame/HudItem.h

```diff
@@ -185,5 +185,7 @@ class CHudItem : public CHUDState
     virtual CHudItem* cast_hud_item() { return this; }
     void PlayAnimIdleMovingCrouch(); //AVO: new crouch idle animation
     bool isHUDAnimationExist(pcstr anim_name) const;
+    bool isHUDAnimationExist(pcstr anim_name, bool is_silent) const;
     pcstr WhichHUDAnimationExist(pcstr anim_name, pcstr anim_name2) const;
+    pcstr WhichHUDAnimationExist(pcstr anim_name, pcstr anim_name2, bool is_silent) const;
```

Comment:
This is better IMO:
```suggestion
    bool isHUDAnimationExist(pcstr anim_name, bool silent = false) const;
    pcstr WhichHUDAnimationExist(pcstr anim_name, pcstr anim_name2, bool silent = false) const;
```

Also notice that `is_silent` was renamed to `silent`. Not too much need in using `is` here.

Reply:
Will update for all methods